lib.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-use-before-define */
+const myLibrary = [];
+const books = document.getElementById('book_list');
+
+const form = document.getElementById('forms');
+const read1 = document.getElementById('read1');
+
+const newBook = document.getElementById('addBook');
+
+function Book(name, author, pages, read) {
+  this.name = name;
+  this.author = author;
+  this.pages = pages;
+  this.read = read;
+}
+
+function deletethisBook(i, myLibrary) {
+  myLibrary.splice(i, 1);
+  displayLibrary(myLibrary);
+}
+
+function changeStatus(i, myLibrary) {
+  myLibrary[i].read = !myLibrary[i].read;
+  displayLibrary(myLibrary);
+}
+
+function displayLibrary(myLibrary) {
+  books.innerHTML = '';
+  for (let i = 0; i < myLibrary.length; i += 1) {
+    const para = document.createElement('div');
+    const currentDiv = document.getElementById('book_list');
+    const currentTable = myLibrary[i];
+    para.innerHTML = `${currentTable.name} ${currentTable.author}
+          ${currentTable.pages}
+          ${currentTable.read} <button onclick = ${deletethisBook(i, myLibrary)}> Delete </button>
+          <button onclick = ${changeStatus(i, myLibrary)}> Change Status</button>
+          `;
+    currentDiv.appendChild(para);
+  }
+}
+
```

Comment:
- Due to the bug above the display book function is not working. The following functionalities should be working ⬇️ 

- [ ] It should display on the page every book found in your array.  ⬇️ 
![render_requirement](https://user-images.githubusercontent.com/32361963/128158688-35419a3a-e3b0-4f37-9f3e-1692d69e3744.png)

- [ ] Each Book displayed has a button to remove the Book from the Library array ⬇️ 
![remove_requirement](https://user-images.githubusercontent.com/32361963/128158805-de6f9b36-29bb-46bc-957b-4199a4a827c2.png)

- [ ] Each `Book` displayed has a `button` to `toggle` the `Book` read status ⬇️ 
![toggle_requirement](https://user-images.githubusercontent.com/32361963/128158953-ebec862f-7f24-45d7-8977-8f1e1a5fea04.png)